SECURITY.md

```diff
@@ -0,0 +1,41 @@
+# Osmosis Bug Reporting and Feature Requests
+
+Osmosis uses GitHub to manage feature requests and bugs for its platform. This is done via GitHub Issues.
+
+## Feature Request
+
+For a feature request, please [create a GitHub issue](https://github.com/osmosis-labs/osmosis/issues/new). Clearly state your use case and what value it will bring to other users or developers on Osmosis.
+
+If it is something that can be handled by a parameter change, discuss it on [the discussions](https://github.com/osmosis-labs/osmosis/discussions) or on [Discord](https://discord.com/invite/osmosis) in `#governance-proposals` and consider a governance proposal.
```

Comment:
Should these be on github or commmonwealth? cc (cc @johnnywyles)

Reply:
> @daniel-farina are you able to resolve comments and then we can get this merged?

This one got lost in time. Solving comments and merging 👍

Reply:
updated 👍